netutils/config/parser.py

```diff
@@ -441,17 +442,18 @@ def _build_banner(self, config_line):
         Raises:
             ValueError: When the parser is unable to identify the End of the Banner.
         """
-        if self.is_banner_one_line(config_line):
+        if CiscoConfigParser.is_banner_one_line(config_line):
```

Comment:
why this change?

Reply:
Thought it would be clearer that the call was a `staticmethod` call.

Reply:
I think it reads that the current code block is outside of the `CiscoConfigParser` class

Reply:
No probs, will go with `self`